src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -303,6 +303,31 @@ private static long FileTimeToPosix(FILETIME ft)
             return date / 10000000;
         }
 
+        private IDebugMemoryContext2 GetMemoryContext(string memoryReference, int? offset, out ulong address, out int hr)
```

Comment:
nit: Normally we return the `hr` as the result of the function and make the object we want an `out` param.